config/currency_iso.yml

```diff
@@ -1446,19 +1446,20 @@ mop:
   thousands_separator: ","
   iso_numeric: '446'
   smallest_denomination: 10
-mro:
+mru:
   priority: 100
-  iso_code: MRO
-  name: Mauritanian Ouguiya
+  iso_code: MRU
+  name: Mauritanian New Ouguiya
   symbol: UM
+  disambiguate_symbol: MRU
   alternate_symbols: []
   subunit: Khoums
-  subunit_to_unit: 5
+  subunit_to_unit: 1
```

Comment:
It might be worthwhile to have a comment explaining the "no subunit" thing, like there is below for ves' `smallest_denomination`, in case there are other changes and the git blame doesn't clearly point to your helpful commit message.